Resources/Prototypes/_Impstation/Catalog/Cargo/cargo_service.yml

```diff
@@ -19,6 +19,15 @@
   group: market
 
 - type: cargoProduct
+  id: ServiceBarKit
+  icon:
```

Comment:
this icon has a bit too much empty space around it compared to other cargo products: is there another one you could use?